code/modules/clothing/suits/storage.dm

```diff
@@ -21,6 +21,12 @@
 	if (pockets.handle_mousedrop(usr, over_object))
 		..(over_object)
 
+/obj/item/clothing/suit/storage/handle_middle_mouse_click(mob/user)
+	if(Adjacent(user))
```

Comment:
`use_check_and_message()`?

Reply:
i just copied how everything else did it, presumably the best place for the checks would be in the open or handle_middle_mouse_click caller themselves